### PR TITLE
GRF-512 : Load the template data

### PR DESCRIPTION
**Pull request by jeremyBeaucousinAkeneo**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  });
+
+  const labelsFields: ReactNode[] = sortedLocaleCodes.map(function (localeCode) {
+    const localeName = findLocaleName(localeCode);
+    const value = template.labels[localeCode] || '';
+
+    return (
+      <Field label={localeName} key={localeCode}>
+        <TextInput
+          //name={`${labelField}-${locale}`}
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx:69]: useless comments?

File: src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+const FormContainer = styled.div`
+  margin-top: 20px;
+
+  & > * {
+    margin: 0 10px 20px 0;
+  }
+`;
+
+// const ErrorMessage = styled(Helper)`
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx:21]: useless?

File: src/Akeneo/Category/front/src/feature/pages/TemplatePage.tsx [hunk 3]
```diff
@@ -92,7 +137,7 @@ const TemplateIndex: FC = () => {
             className="AknTitleContainer-userMenuContainer AknTitleContainer-userMenu"
           />
         </PageHeader.UserActions>
-        <PageHeader.Title>templateLabel</PageHeader.Title>
+        <PageHeader.Title>{templateLabel ?? templateId}</PageHeader.Title>
```
Comment (BryanS29-akeneo) [inline src/Akeneo/Category/front/src/feature/pages/TemplatePage.tsx:140]: Why display templateId?

File: src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx (hunk 1 above)
Comment (etiennekerbastard) [reply src/Akeneo/Category/front/src/feature/components/template/EditTemplatePropertiesForm.tsx:69]: indeed

File: src/Akeneo/Category/front/src/feature/pages/TemplatePage.tsx (hunk 3 above)
Comment (jeremyBeaucousinAkeneo) [reply src/Akeneo/Category/front/src/feature/pages/TemplatePage.tsx:140]: We displayed it only if the label is not found

**Pull request merged**